include/openmc/error.h

```diff
@@ -63,6 +66,13 @@ void write_message(const std::stringstream& message, int level)
   write_message(message.str(), level);
 }
 
+template <typename...Params>
+void format_write_message(const int level, const std::string& fmt_msg, const Params&...fmt_args) {
```

Comment:
`const` is superfluous on an integer passed by value
```suggestion
void format_write_message(int level, const std::string& fmt_msg, const Params&...fmt_args) {
```

Reply:
Thanks for this insight. I've added it to the updated version and will be pushing soon